website/content/v1.6/talos-guides/howto/workers-on-controlplane.md

```diff
@@ -16,15 +16,4 @@ cluster:
     allowSchedulingOnControlPlanes: true
 ```
 
-This may be done via editing the `controlplane.yaml` file before it is applied to the controlplane nodes, by `talosctl edit machineconfig`, or by [patching the machine config]({{< relref "../configuration/patching">}}).
-
-> Note: if you edit or patch the machine config on a running control plane node to set `allowSchedulingOnControlPlanes: true`, it will be applied immediately, but will not have any effect until the next reboot.
-You may reboot the nodes via `talosctl reboot`.
-
-You may also immediately make the control plane nodes schedulable by running the below:
-
-```bash
-kubectl taint nodes --all  node-role.kubernetes.io/control-plane-
-```
-
-Note that unless `allowSchedulingOnControlPlanes: true` is set in the machine config, the nodes will be tainted again on next reboot.
+This may be done via editing the `controlplane.yaml` file before it is applied to the controlplane nodes, by [editing the machine config]({{< relref "../configuration/editing-machine-configuration" >}}), or by [patching the machine config]({{< relref "../configuration/patching">}}).
```

Comment:
```suggestion
This may be done via editing the `controlplane.yaml` file before it is applied to the control plane nodes, by [editing the machine config]({{< relref "../configuration/editing-machine-configuration" >}}), or by [patching the machine config]({{< relref "../configuration/patching">}}).
```